rust/libnewsboat/src/utils.rs

```diff
@@ -311,6 +312,26 @@ pub fn is_valid_podcast_type(mimetype: &str) -> bool {
     matches || found
 }
 
+pub fn get_auth_method(method: &str) -> u64 {
+    match method {
+        "basic" => curl_sys::CURLAUTH_BASIC,
+        "digest" => curl_sys::CURLAUTH_DIGEST,
+        "digest_ie" => curl_sys::CURLAUTH_DIGEST_IE,
+        "gssnegotiate" => curl_sys::CURLAUTH_GSSNEGOTIATE,
+        "ntlm" => curl_sys::CURLAUTH_NTLM,
+        "anysafe" => curl_sys::CURLAUTH_ANYSAFE,
+        "any" | "" => curl_sys::CURLAUTH_ANY,
+        _ => {
+            log!(
+                Level::UserError,
+                "utils::get_auth_method: you configured an invalid proxy type: {}",
```

Comment:
Please change the string to what it said in C++: "you configured an invalid proxy authentication method". "Proxy type" is not the same as "proxy authentication method".

Reply:
Ops, I forgot to change that.